tests/integration/dev.bats

```diff
@@ -75,6 +75,12 @@ function teardown() {
 	# test read
 	runc exec test_allow_char sh -c 'head -n 1 /dev/kmsg'
 	[ "$status" -eq 0 ]
+
+	# test access
+	TEST_NAME="dev_access_test"
+	gcc -static -o "rootfs/bin/${TEST_NAME}" "${TESTDATA}/${TEST_NAME}.c"
+	runc exec test_allow_char sh -c "${TEST_NAME} /dev/kmsg"
```

Comment:
probably no need to wrap this into `sh -c`, iow
```suggestion
	runc exec test_allow_char "${TEST_NAME}" /dev/kmsg
```

will work just fine.

Reply:
Not important though

Reply:
Agree. I just followed the same approach as in other tests here. So used `sh -c` rather for consistency across this file.